src/styles/index.scss

```diff
@@ -1,3 +1,156 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+  --gray-elements: #f3f3f3;
+  --white: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid var(--gray-elements);
+  background: var(--white);
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__product-img {
+    width: 100%;
+    height: 134px;
+    margin: 0 3px;
+  }
+
+  &__product-link {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-title {
+    color: var(--text-accent-color);
+    width: 100%;
+    margin-top: 40px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-decoration: none;
+  }
+
+  &__product-code {
+    color: var(--text-secondary-color);
+    margin: 0;
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__review {
+    width: 100%;
+    margin-top: 16px;
+
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 18px;
+    margin-top: 24px;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding